### PR TITLE
Cost effective auto purchase for Click/DPS skills

### PR DESCRIPTION
**Pull request by rossengeorgiev**
Continuation of #143

Adds two new toggle options.

### CODE REVIEW

File: autoPlay.user.js [hunk 1]
```diff
@@ -21,9 +21,12 @@ var logLevel = 1; // 5 is the most verbose, 0 disables all log
...
 var enableAutoClicker = getPreferenceBoolean("enableAutoClicker", true);
 
+var enableAutoUpgradeDPS = getPreferenceBoolean("enableAutoUpgradeDPS", true);
+var enableAutoUpgradeClick = getPreferenceBoolean("enableAutoUpgradeClick", true);
+
 var removeInterface = getPreferenceBoolean("removeInterface", true); // get rid of a bunch of pointless DOM
-var removeParticles = getPreferenceBoolean("removeParticles", true);
-var removeFlinching = getPreferenceBoolean("removeFlinching", true);
+var removeParticles = getPreferenceBoolean("removeParticles", false);
+var removeFlinching = getPreferenceBoolean("removeFlinching", false);
```
Comment (iakat) [inline autoPlay.user.js]: Don't touch this.

Comment (xPaw) [inline autoPlay.user.js]: unrelated change

**Comment by rossengeorgiev** [comment]
That change was not intended. Fixed.

File: autoPlay.user.js [hunk 2]
```diff
@@ -346,6 +353,69 @@ function MainLoop() {
 	}
 }
 
+function useAutoUpgrade() {
+	if(!enableAutoUpgradeDPS && !enableAutoUpgradeClick) { return; }
```
Comment (chauffer) [inline autoPlay.user.js]: Why have two checkboxes if you check both here?

Comment (rossengeorgiev) [inline autoPlay.user.js]: for quite exit of the function, if both are disabled
